Source/WebKit/UIProcess/dmabuf/AcceleratedBackingStoreDMABuf.messages.in

```diff
@@ -24,5 +24,5 @@ messages -> AcceleratedBackingStoreDMABuf NotRefCounted {
     DidCreateBuffer(uint64_t id, WebCore::IntSize size, uint32_t format, Vector<UnixFileDescriptor> fds, Vector<uint32_t> offsets, Vector<uint32_t> strides, uint64_t modifier, WebKit::DMABufRendererBufferFormat::Usage usage)
     DidCreateBufferSHM(uint64_t id, WebCore::ShareableBitmapHandle handle)
     DidDestroyBuffer(uint64_t id)
-    Frame(uint64_t id)
+    Frame(uint64_t id, WebCore::Damage damage)
```

Comment:
I think we could just pass a `std::optional<WebCore::Region>` and we don't need a serializer for the damage.

Reply:
Will do, and remove the `Damage.serialization.in` file along with the change.